spring-cloud-gcp-samples/spring-cloud-gcp-trace-sample/src/test/resources/logback-test.xml

```diff
@@ -10,8 +10,13 @@
 		</encoder>
 	</appender>
 
+	<appender name="ASYNC" class="ch.qos.logback.classic.AsyncAppender">
+		<appender-ref ref="STACKDRIVER" />
+	</appender>
+
 	<root level="INFO">
 		<appender-ref ref="STDOUT" />
-		<appender-ref ref="STACKDRIVER" />
+<!--		<appender-ref ref="STACKDRIVER" />-->
```

Comment:
Meant to comment out or remove?

Reply:
Sorry, wasn't really ready for review. Needed tests.